Modix.Data/Models/Tags/TagSummary.cs

```diff
@@ -71,10 +71,10 @@ public class TagSummary
                 Uses = entity.Uses,
                 OwnerUser = (entity.OwnerUserId == null)
                     ? null
-                    : entity.OwnerUser.Project(GuildUserBrief.FromEntityProjection),
+                    : entity.OwnerUser.Project(GuildUserBrief.FromEntityProjection!),
```

Comment:
What the hell? How does the compiler consider this nullable?

Reply:
It's a delegate type mismatch. `Project` says it will pass a `null` `GuildUser`, but we've just checked the id so we know that `OwnerUser` is not null and this is safe to call.

Reply:
That would imply to me that the change should be
```cs
entity.OwnerUser!.Project(GuildUserBrief.FromEntityProjection)
```

Doesn't the `!` annotation inside the `Project()` call imply that you're saying the `Expression` object is not null?

Reply:
Well, if we change to comparing `OwnerUser` for `null` as you suggested above, then the need for the `!` goes away entirely. And yes, that is what that's implying. But that's not where the warning is occurring.

Reply:
I just..... I don't see how this whole thing isn't a compiler bug.

It really doesn't matter, whether or not it is has nothing to do with this PR. I just don't get it.